pimd/pim_nht.c

```diff
@@ -133,7 +133,7 @@ static struct pim_nexthop_cache *pim_nht_get(struct pim_instance *pim,
 /* TBD: this does several distinct things and should probably be split up.
  * (checking state vs. returning pnc vs. adding upstream vs. adding rp)
  */
-int pim_find_or_track_nexthop(struct pim_instance *pim, struct prefix *addr,
+int pim_find_or_track_nexthop(struct pim_instance *pim, pim_addr *addr,
```

Comment:
Same here, consistent for `pim_addr` instead of `pim_addr *`

Reply:
Fixed